### PR TITLE
Clean up README

### PR DESCRIPTION
**Pull request by tomdaley92**
Made the following changes:

- windows installation uses a python virtual environment now 
    - @ScottJDaley can  you test that this works on windows?
- MacOS installation steps now includes `homebrew`
- All OS-installation steps now include installing Python3 and Graphiz, and Git, and Cloning the repository 
- Remove Windows-specific usage from `Command Line Tool` section in README
- Improve formatting of the commands

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,138 +99,155 @@ outputs, and adhere to any given constraints.
...
+2. Invite the bot to your server using a link generated from OAuth2 page on https://discord.com/developers/applications.
+3. The bot should be able to respond to commands in Discord after a few seconds.
+4. Press enter in the terminal to shut down the bot.
+
+#### Command line tool
+1. Run `tool.py`.
 2. Type a query and press Enter.
 3. Type `exit` to quit.
 
-#### Discord bot
```
Comment (tomdaley92) [inline README.md:220]: @ScottJDaley 

I wasn't exactly sure what was going on with this section, but it felt like is overlapped with installation instructions a bit and had Windows specific functionality so I changed it up.

File: README.md [hunk 2]
```diff
@@ -99,138 +99,155 @@ outputs, and adhere to any given constraints.
...
-#### Command line tool
 
-1. Double click `tool.py` to run it.
+#### Usage
+1. Ensure the bot is running and was started with a valid `DISCORD_TOKEN` as described above.
+2. Invite the bot to your server using a link generated from OAuth2 page on https://discord.com/developers/applications.
+3. The bot should be able to respond to commands in Discord after a few seconds.
+4. Press enter in the terminal to shut down the bot.
+
+#### Command line tool
```
Comment (tomdaley92) [inline README.md:246]: I might have goofed up here. I don't actually know what `tool.py` is or if it is still valid. Is it just a convenience script for running the bot a different way?

Comment (tomdaley92) [inline README.md]: Is this section for testing it without Discord? Double check I didn't goof anything up by changing the documentation a bit

**Comment by tomdaley92** [comment]
FYI I set the base branch as the IaC Update feature branch. I guess I could have just done it against the development branch, but doesn't really matter I guess

File: README.md [hunk 3]
```diff
@@ -99,138 +99,157 @@ outputs, and adhere to any given constraints.
...
-cd ada
-```
+   ```cmd
+   cd ada
+   ```
+
+5. Create virtual environment and activate it
+
+   ```cmd
+   py -m venv venv && venv\Scripts\activate
```
Comment (tomdaley92) [inline README.md]: @ScottJDaley Can you test this syntax on Windows? It should work with both CMD and powershell.

File: README.md (hunk 1 above)
Comment (ScottJDaley) [reply README.md:220]: Yeah, so there are two ways to use this. One is as a bot through discord. The other is through the command line.

File: README.md (hunk 2 above)
Comment (ScottJDaley) [reply README.md]: Yeah exactly. I'm fine with making a "Usage" section that is entirely about the bot. The command line option is more for debugging and I never use myself anyways. 

However, I think this needs to be a heading 3 (###). Right now it looks like a sub item of the Linux Installation.

File: README.md (hunk 3 above)
Comment (ScottJDaley) [reply README.md]: works in cmd:
image

Not working in powershell:
image

Comment (ScottJDaley) [reply README.md]: This works in powershell though:
image

File: README.md [hunk 4]
```diff
@@ -99,138 +99,157 @@ outputs, and adhere to any given constraints.
  - Add install directory `C:\Program Files (x86)\Graphviz2.38\bin` to PATH
  
 3. Run the following as admin to configure GraphViz:
-```
-dot -c
-```
+   ```
```
Comment (ScottJDaley) [inline README.md]: should this be "```cmd"?

File: README.md [hunk 5]
```diff
@@ -99,138 +99,157 @@ outputs, and adhere to any given constraints.
...
+   cd ada
+   ```
+
+5. Create virtual environment and activate it
+
+   ```cmd
+   py -m venv venv && venv\Scripts\activate
+   ```
+
+   :bulb: You may deactivate the virtual environment with `deactivate`
```
Comment (ScottJDaley) [inline README.md]: lol, I just had to google this

File: README.md [hunk 6]
```diff
@@ -99,138 +99,157 @@ outputs, and adhere to any given constraints.
...
-
-6. Start the bot
-```bash
-python bot.py
-```
+   ```zsh
+   brew install graphviz
+   ```
+
+1. Clone repo (in directory of you choosing)
```
Comment (ScottJDaley) [inline README.md]: The numbering here keeps restarting. Pretty sure you can just do 1. for all the items in a markdown list.

File: README.md (hunk 3 above)
Comment (ScottJDaley) [reply README.md]: How about we split this up into two steps so that it is terminal-agnostic?

1. `py -m venv venv`
2. `venv\Scripts\activate`

Comment (ScottJDaley) [reply README.md]: Done.

**Pull request merged**